test.py

```diff
@@ -2,13 +2,25 @@
 """
 Some basic tests to verify that the wrapper is working
 """
+from os import getenv
+
+from dotenv import load_dotenv, find_dotenv
```

Comment:
Nope, I missed that as well. I imagine the test.py was just someone's way of writing their own "do this to make sure it works" rather than automating anything.